docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -15,7 +15,7 @@ This document describes the current state of Horizontal Pod Autoscaling in Kuber
 
 With Horizontal Pod Autoscaling, Kubernetes automatically scales the number of pods
 in a replication controller, deployment or replica set based on observed CPU utilization
-(or, with alpha support, on some other, application-provided metrics).
+(or, with (custom metrics)[https://github.com/kubernetes/community/blob/master/contributors/design-proposals/custom-metrics-api.md] support, on some other, application-provided metrics).
```

Comment:
* Parenthesis are wrong - name should be in [] and link in ()
* why do you remove information about this feature being alpha?

Reply:
should be http://git.k8s.io/community/contributors/design-proposals/instrumentation/custom-metrics-api.md now